packages/api/src/transport/ws/ws.js

```diff
@@ -261,7 +261,14 @@ class Ws extends JsonRpcBase {
 
         // Don't print error if request rejected or not is not yet up...
         if (!/(rejected|not yet up)/.test(result.error.message)) {
-          console.error(`${method}(${JSON.stringify(params)}): ${result.error.code}: ${result.error.message}`);
+          const dangerous_methods = ['signer_confirmRequest', 'signer_confirmRequestWithToken'];
+          let safe_params;
+          if (dangerous_methods.includes(method)) {
+            safe_params = params.slice();
+            safe_params[params.length - 1] = '***';
```

Comment:
Let's add a comment somewhere here, to tell future devs what we're doing here, it might not be obvious at 1st sight.